### PR TITLE
Remove kubevirt-web-ui from dependencies

### PR DESCRIPTION
**Pull request by mareklibra**
All code has been gradually migrated into openshift/console code
base so there is not reason to keep this dependency.

### CODE REVIEW

**Comment by mareklibra** [comment]
Rebased and `yarn.lock` modification moved to separate commit

File: frontend/packages/kubevirt-plugin/src/style.scss [hunk 1]
```diff
@@ -6,7 +6,6 @@
 @import "~patternfly/dist/sass/patternfly/variables";
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/style.scss]: these are only the dependencies neeeded for the react-console, right? It is probably the only component which uses pf3.

are all of these imports needed?

File: frontend/packages/kubevirt-plugin/src/style.scss [hunk 2]
```diff
@@ -6,7 +6,6 @@
...
 @import "~patternfly/dist/sass/patternfly/bootstrap-mixin-overrides";
 @import "~patternfly/dist/sass/patternfly/mixins";
-@import '~patternfly/dist/sass/patternfly/icons'; // TODO remove together with kubevirt-web-ui-components dependency
 @import "~patternfly-react/dist/sass/patternfly-react";
 
 @import '~patternfly/dist/sass/patternfly/wizard';
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/style.scss]: this one can be removed

Comment (mareklibra) [reply frontend/packages/kubevirt-plugin/src/style.scss]: ok

File: frontend/packages/kubevirt-plugin/src/style.scss (hunk 1 above)
Comment (mareklibra) [reply frontend/packages/kubevirt-plugin/src/style.scss]: I tried to remove them but they appeared to be needed by console

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/src/style.scss]: ok, can you please write a comment there describing these dependencies? So people know why they are here in the future.

**Pull request merged**